modules/ROOT/pages/architecture.adoc

```diff
@@ -42,6 +42,32 @@ The minimum requirements are 3 controller and 3 worker nodes. 3 controllers enab
 
 Mule applications run on workers. Multiple replicas of applications can run across workers.
 
+== Scaling Considerations
+
+* Required number of controller nodes. The maximum number of controller nodes supported is 3.
+
+** Production environment: 3
```

Comment:
Should clarify these are minimum requirements.

Reply:
Updated this - please verify.